com.unity.render-pipelines.core/Runtime/Volume/Volume.cs

```diff
@@ -20,7 +20,7 @@ public class Volume : MonoBehaviour
         /// The Volume priority in the stack. A higher value means higher priority. This supports negative values.
         /// </summary>
         [Tooltip("Sets the Volume priority in the stack. A higher value means higher priority. You can use negative values.")]
-        public float priority = 0f;
+        public int priority = 0;
```

Comment:
NOTE: This is in pipeline.core. Might affect HDRP.

Reply:
What's the reason to change this? I understand it makes a lot more sense as an Int but this Quality Audit is targeting our 2021.2 release so it feels a bit unnecessary to change this so late.

Reply:
........ if its low stress..... i'm on board with changing it. I think this item is also in the quality audit

Reply:
Yes, it's one of the items in the list.
This might cause some breakage which is why I flagged this.
I didn't see any issues for URP with minimal testing, but this should be go through QA.

Reply:
Looks like this will break (CinemachineVolumeSettings.cs(266,34)) because float/int assignment.
Revert?

Reply:
The problem though is that this is a public API breaking change.
Which means we'll have to do our normal deprecation dance here (Warning->Error->Remove). 
I propose we postpone this change because of that.

Reply:
Reverted.